app/insomnia/enterprise-onboarding.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Get started with Insomnia Enterprise
+
+description: Learn how to fully set up your Insomnia Enterprise instance.
+
+content_type: reference
+layout: reference
+breadcrumbs: 
+  - /insomnia/
+  - /insomnia/enterprise/
+products:
+    - insomnia
+tier: enterprise
+related_resources:
+  - text: Enterprise
+    url: /insomnia/enterprise/
+  - text: Enterprise account management
+    url: /insomnia/enterprise-account-management/
+  - text: Enterprise user management
+    url: /insomnia/enterprise-user-management/
+
+next_steps:
+  - text: Documents
+    url: /insomnia/documents/
+  - text: Collections
+    url: /insomnia/collections/
+  - text: Environments
+    url: /insomnia/environments/
+  - text: Mock servers
+    url: /insomnia/mock-servers/
+  - text: Scripts
+    url: /insomnia/scripts/
+
+---
+
+If you're new to Insomnia Enterprise, this document will guide you through the full setup of your instance in [**Insomnia Admin**](https://app.insomnia.rest/).
+
+## 1. Activate your Enterprise membership
+
+Once you've signed up for the Insomnia Enterprise plan through the [sales team](https://insomnia.rest/pricing/contact), you'll get an activation code.
+The first thing you'll need to do is:
+
+1. Go to [Change subscription plan](https://app.insomnia.rest/app/subscription/update).
```

Comment:
This link gave me a 404. Can we also provide the UI instructions for how they can get to this location in their dashboard?